src/dsp_tools/utils/connection_live.py

```diff
@@ -240,7 +240,7 @@ def post(
         if not route.startswith("/"):
             route = f"/{route}"
         url = self.server + route
-        headers = {}
+        headers = {"X-Asset-Ingested": "true"}
```

Comment:
If you change that here, every single POST request that DSP-TOOLS makes, will have this header. I doubt that this is what we want... We should only send this header if a resource is created via ingest-xmlupload. That means, we have to parametrize this. I hope it won't be too much work, but I fear we cannot just leave it as it is.